pulsar-client-cpp/lib/TopicName.cc

```diff
@@ -52,8 +52,19 @@ TopicName::TopicName() {}
 bool TopicName::init(const std::string& topicName) {
     topicName_ = topicName;
     if (topicName.find("://") == std::string::npos) {
-        LOG_ERROR("Topic name is not valid, domain not present - " << topicName);
-        return false;
+        std::string topicNameCopy_ = topicName;
+        std::vector<std::string> pathTokens;
+        boost::algorithm::split(pathTokens, topicNameCopy_, boost::algorithm::is_any_of("/"));
+        if (pathTokens.size() == 3) {
+          topicName_ = "persistent://" + pathTokens[0] + "/" + pathTokens[1] + "/" + pathTokens[2];
```

Comment:
This will probably break the format checker that expects 4 spaces indent

Reply:
nice catch.